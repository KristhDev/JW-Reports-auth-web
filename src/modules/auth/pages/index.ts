export { Confirm } from './Confirm';
export { ResetPassword } from './ResetPassword';