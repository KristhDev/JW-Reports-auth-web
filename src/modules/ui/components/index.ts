export { Button } from './Button';
export { EyeBtn } from './EyeBtn';
export { FormField } from './FormField';
export { Spinner } from './Spinner';
export { Title } from './Title';