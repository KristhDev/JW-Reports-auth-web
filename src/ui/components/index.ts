export { FormField } from './FormField';
export { Title } from './Title';