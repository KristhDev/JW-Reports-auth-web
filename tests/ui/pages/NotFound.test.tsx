import { render } from '@testing-library/react';

import { NotFound } from '../../../src/ui/pages';

describe('Test in <NotFound /> page', () => {
    it('should to match the snapshot', () => {
        const { container } = render(<NotFound />);
        expect(container).toMatchSnapshot();
    });
});