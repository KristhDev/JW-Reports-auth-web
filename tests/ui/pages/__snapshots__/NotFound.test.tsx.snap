// Vitest Snapshot v1

exports[`Test in <NotFound /> page > should to match the snapshot 1`] = `
<div>
  <div
    class="container center-content"
  >
    <h2>
      No encontrado
    </h2>
    <h1>
      404
    </h1>
    <h3>
      Lo sentimos pero no se encuentra la p√°gina solicitada
    </h3>
  </div>
</div>
`;
